doc/man3/EVP_PKEY_CTX_get0_pkey.pod

```diff
@@ -0,0 +1,53 @@
+=pod
+
+=head1 NAME
+
+EVP_PKEY_CTX_get0_pkey,
+EVP_PKEY_CTX_get0_peerkey
+- functions for accessing the B<EVP_PKEY> associated with an EVP_PKEY_CTX
+
+=head1 SYNOPSIS
+
+ #include <openssl/evp.h>
+
+ EVP_PKEY *EVP_PKEY_CTX_get0_pkey(EVP_PKEY_CTX *ctx);
+ EVP_PKEY *EVP_PKEY_CTX_get0_peerkey(EVP_PKEY_CTX *ctx);
+
+=head1 DESCRIPTION
+
+EVP_PKEY_CTX_get0_pkey() is used to access the EVP_PKEY I<pkey>
+associated with the given EVP_PKEY_CTX I<ctx>.
+The EVP_PKEY obtained is the same one used for creating the EVP_PKEY_CTX
+using either L<EVP_PKEY_CTX_new(3)> or L<EVP_PKEY_CTX_new_from_pkey(3)>.
+
+EVP_PKEY_CTX_get0_peerkey() is used to access the EVP_PKEY I<peerkey>
+associated with the given EVP_PKEY_CTX I<ctx>.
+The EVP_PKEY obtained is the same one used for setting the EVP_PKEY_CTX's I<peerkey>
+using either L<EVP_PKEY_derive_set_peer(3)> or L<EVP_PKEY_derive_set_peer_ex(3)>.
+
+=head1 RETURN VALUES
+
+EVP_PKEY_CTX_get0_pkey() returns the EVP_PKEY I<pkey> associated with the
+EVP_PKEY_CTX or NULL if it is not set.
+
+EVP_PKEY_CTX_get0_peerkey() returns the EVP_PKEY I<peerkey> associated with the
+EVP_PKEY_CTX or NULL if it is not set.
+
+For both functions, I<the returned value should not be freed by the caller>.
+These merely act as Getter functions, and should be treated as such.
```

Comment:
I'd rather replace this sentence with something like: `The returned EVP_PKEY objects are owned by the EVP_PKEY_CTX.` Or at least use lowercase `getter`.

Reply:
Something like:-

```
The returned EVP_PKEY objects are owned by the EVP_PKEY_CTX,
and therefore should not explicitly be freed by the caller.

These functions do not affect the EVP_PKEY reference count.
They merely act as getter functions, and should be treated as such.
```